common/utils.py

```diff
@@ -48,3 +50,10 @@ def get_date(day_arg):
     """
     day_num = _get_day_arg(day_arg)
     return date.today() + timedelta(days=day_num)
+
+
+def render(filename, context={}):
+    path = 'templates'
+    return jinja2.Environment(
+        loader=jinja2.FileSystemLoader(path)
```

Comment:
We might need nested folders in the template in the future, I suggest we pass the `path` as a parameter with  `templates` as the default value.
```
def render(filename, path='templates', contex={}):
   pass
```